vtr_flow/benchmarks/hdl_include/include/README.md

```diff
@@ -0,0 +1,44 @@
+# include files
+
+This folder contains _include_ files for running the modified version of ch_intrisinc.
+
+_include_ files can be either a Verilog file or a Verilog header file. The main point worth mentioning is that the union of include files and the circuit design includes the top module should not result in any conflict like having multiple top modules or declaring different variables with the same name.
+
+To create a task config file, the syntax for _include_ files is pretty much like the circuits or architectures. 
+In the beginning, _inlcudes_dir_, a path to _include_ files should be specified. In the following, specifiers include_add_list adds specific _include_ files, using a relative path that will be pre-pended by the _includes_dir_ path.
```

Comment:
Typo:  inlcudes_dir : should be "includes_dir

Reply:
I forgot to save this file!